src/docfx/config/ops/OpsPreProcessor.cs

```diff
@@ -52,7 +52,13 @@ private void PreProcessMonoDocXml()
                         Path.Combine(_buildOptions.DocsetPath, _config.Monodoc.SourceXmlFolder),
                         Path.Combine(_buildOptions.DocsetPath, _config.Monodoc.OutputYamlFolder),
                         fallbackXmlPath,
-                        item => _errorLog.Write(ConstructError(item)),
+                        item =>
```

Comment:
nit: consider extract the logic into a method like `void Foo(LogItem)` so there is less logic here. Think if there is powershell and we also need to pass an `Action<LogItem>`
```suggestion
                        WriteError,
```